lib/myhttp.js

```diff
@@ -29,6 +29,7 @@ function Client (opts) {
   this.bytes = 0
   this.headers = {}
   this.startTime = [0, 0]
+  this.body = this.opts.body
```

Comment:
I'm not sure, as this will allow the user to assign to `client.body` and when the `rebuildRequest()` is called, it should pick up their assignments. However, I would prefer to take this out and just use `.setBody(...)` so its is a consistent api for headers and body.